### ISSUE TITLE
[Feature Request] Do expert review / Usability / Accessibility / UX

### ISSUE THREAD

**Issue by yayitserica** [microsoft/vsts-extension-retrospectives#41]
To discuss on Fri 7/17.

As a designer I want to do an expert review, so that we can start identifying the key components to focus on, considering:

1. Importance to user = i.e. user needs a timer
2. Technical feasibility = i.e. timer might be a 1/10 technical complexity
3. Design/process feasibility = i.e. timer might be a 10/10 in terms of design/process (who uses the timer?  Who has permissions to the timer?)

**Comment by yayitserica** [comment]
@polatengin , I'm not able to assign @JankaKoen to this issue.  Do you know how to do this?

**Comment by polatengin** [comment]
something is different with @JankaKoen s GitHub account. when I tag her, her name doesn't show up in the list, but I can see I tagged her when I Preview my message.

and definitely her name doesn't show up in the Assignees list, so I can't do that too 😀

**Comment by yayitserica** [comment]
To take place Mon 7/27

**Comment by polatengin** [comment]
I think I added this feature long time ago, please feel free to reopen it if you think something is missing
